server/monitor/src/main/resources/org/apache/accumulo/monitor/resources/external/flot/jquery.flot.time.js

```diff
@@ -248,25 +248,25 @@ API.txt for details.
         "microsecond": 0.001,
         "millisecond": 1,
         "second": 1000,
-        "minute": 60 * 1000,
-        "hour": 60 * 60 * 1000,
-        "day": 24 * 60 * 60 * 1000,
-        "month": 30 * 24 * 60 * 60 * 1000,
-        "quarter": 3 * 30 * 24 * 60 * 60 * 1000,
-        "year": 365.2425 * 24 * 60 * 60 * 1000
+        "minute": 60_000,
+        "hour": 60 * 60_000,
+        "day": 24 * 60 * 60_000,
+        "month": 30 * 24 * 60 * 60_000,
+        "quarter": 3 * 30 * 24 * 60 * 60_000,
+        "year": 365.2425 * 24 * 60 * 60_000
     };
 
     // map of app. size of time units in microseconds
     var timeUnitSizeMicroseconds = {
         "microsecond": 1,
         "millisecond": 1000,
         "second": 1000000,
-        "minute": 60 * 1000000,
-        "hour": 60 * 60 * 1000000,
-        "day": 24 * 60 * 60 * 1000000,
-        "month": 30 * 24 * 60 * 60 * 1000000,
-        "quarter": 3 * 30 * 24 * 60 * 60 * 1000000,
-        "year": 365.2425 * 24 * 60 * 60 * 1000000
+        "minute": 60_000_000,
+        "hour": 60 * 60_000_000,
+        "day": 24 * 60 * 60_000_000,
+        "month": 30 * 24 * 60 * 60_000_000,
+        "quarter": 3 * 30 * 24 * 60 * 60_000_000,
+        "year": 365.2425 * 24 * 60 * 60_000_000
```

Comment:
This file shouldn't be changed, because it's a bundled third-party file and not our own code. We don't want it to diverge from the upstream bytes we got it from.